src/commands/createWebApp/setPostPromptDefaults.ts

```diff
@@ -39,48 +39,74 @@ export async function setPostPromptDefaults(wizardContext: IWebAppWizardContext,
+                            wizardContext.newResourceGroupName = groupName;
+                            wizardContext.newPlanName = plan.name;
+                            break;
+                        }
                     }
                 }
-            }
 
-            wizardContext.newResourceGroupName = wizardContext.newResourceGroupName || await siteNameStep.getRelatedName(wizardContext, defaultGroupName);
-            if (!wizardContext.newResourceGroupName) {
-                throw new Error(localize('noUniqueNameRg', 'Failed to generate unique name for resources. Use advanced creation to manually enter resource names.'));
-            }
+                wizardContext.newResourceGroupName = wizardContext.newResourceGroupName || await siteNameStep.getRelatedName(wizardContext, defaultGroupName);
+                if (!wizardContext.newResourceGroupName) {
+                    throw new Error(localize('noUniqueNameRg', 'Failed to generate unique name for resources. Use advanced creation to manually enter resource names.'));
+                }
 
-            wizardContext.newPlanName = await siteNameStep.getRelatedName(wizardContext, defaultPlanName);
-            if (!wizardContext.newPlanName) {
-                throw new Error(localize('noUniqnueNameAsp', 'Failed to generate unique name for app service plan. Use advanced creation to manually enter plan names.'));
+                wizardContext.newPlanName = await siteNameStep.getRelatedName(wizardContext, defaultPlanName);
+                if (!wizardContext.newPlanName) {
+                    throw new Error(localize('noUniqnueNameAsp', 'Failed to generate unique name for app service plan. Use advanced creation to manually enter plan names.'));
+                }
             }
+        } else {
+            wizardContext.newResourceGroupName = defaultGroupName;
+            wizardContext.newPlanName = defaultPlanName;
+        }
+    } catch (e) {
+        if (parseError(e).errorType === 'AuthorizationFailed') {
+            wizardContext.newResourceGroupName = defaultGroupName;
+            wizardContext.newPlanName = defaultPlanName;
+            setPlanSkuFamilyFilter(wizardContext);
+        } else {
+            throw e;
         }
-    } else {
-        wizardContext.newResourceGroupName = defaultGroupName;
-        wizardContext.newPlanName = defaultPlanName;
+    }
+}
+
+function setPlanSkuFamilyFilter(wizardContext: IWebAppWizardContext): void {
+    switch (nonNullProp(nonNullProp(wizardContext, 'newPlanSku'), 'family')) {
+        case 'F':
+            wizardContext.planSkuFamilyFilter = /^F$/i;
+            break;
+        case 'B':
+            wizardContext.planSkuFamilyFilter = /^B$/i;
+            break;
+        case 'PV2':
+            wizardContext.planSkuFamilyFilter = /^PV2$/i;
+            break;
+        default:
+            break;
```

Comment:
This part is for when  the user doesn't have permission to create a App Service Plan and they are prompted to select from existing. 
By setting the `planSkuFamilyFilter`, only plans matching the user selected tier would be displayed for them.

Reply:
Update: removing this change for this PR as it is not related to #1490.